community-operators/metering/metering-operator.v0.13.0.clusterserviceversion.yaml

```diff
@@ -33,7 +33,7 @@ spec:
     type: OwnNamespace
   - supported: true
     type: SingleNamespace
-  - supported: false
+  - supported: true
```

Comment:
This sets the MultiNamespace installMode to true. Seems that SingleNamespace is already supported.

Reply:
Oh, your right. I totally missed that. Dangit yaml.